### PR TITLE
Upgrading docker version on bastion host - issue #505

### PR DESCRIPTION
**Pull request by ravbaba**
fix for issue #505

### CODE REVIEW

File: terraform/aws/variables.tf [hunk 1]
```diff
@@ -109,3 +109,8 @@ variable "amis" {
     us-west-2 = "ami-00657261"
   }
 }
+
+variable "docker_version" {
+  description = "Docker version"
+  default = "1.8.2-0~trusty"
```
Comment (wallies) [inline terraform/aws/variables.tf]: @ravbaba  can we change this to 1.9

Comment (ravbaba) [inline terraform/aws/variables.tf]: sure, let me test it first

Comment (enxebre) [inline terraform/aws/variables.tf]: ideally should merge this one for the sake of consistency then tag so we got an snapshot with new weave approach plus the same version of docker across the cluster, then create a new pr for new version

Comment (tayzlor) [inline terraform/aws/variables.tf]: Agreed - we could then build the 1.9 into https://github.com/Capgemini/Apollo/pull/520 on top of this change

**Pull request merged**